include/ISceneNode.h

```diff
@@ -273,10 +273,10 @@ namespace scene
 
 		//! Sets the material type of all materials in this scene node to a new material type.
 		/** \param newType New type of material to be set. */
-		void setMaterialType(video::E_MATERIAL_TYPE newType)
+		void setMaterialType(const std::array<video::IGPUSpecializedShader*,5u>& newType)

```

Comment:
kill this function, just let people set .Pipeline directly on materials

Reply:
i think this is quite useful because it sets pipeline on all meshbuffers of mesh scene node